packages/xrpl/test/wallet/index.test.ts

```diff
@@ -437,6 +437,51 @@ describe('Wallet', function () {
     })
   })
 
+  describe('from PrivateKey', function () {
+    describe('using secp256k1 private key', function () {
+      const mockWallet_secp256k1 = {
+        address: 'rhvh5SrgBL5V8oeV9EpDuVszeJSSCEkbPc',
+        publicKey:
+          '030E58CDD076E798C84755590AAF6237CA8FAE821070A59F648B517A30DC6F589D',
+        privateKey:
+          '00141BA006D3363D2FB2785E8DF4E44D3A49908780CB4FB51F6D217C08C021429F',
```

Comment:
Hello, how did you generate this mocked `privateKey` and `publicKey` data? Which library/tool did you use?